modules/33_CDR/portfolio/equations.gms

```diff
@@ -0,0 +1,168 @@
+*** |  (C) 2006-2022 Potsdam Institute for Climate Impact Research (PIK)
+*** |  authors, and contributors see CITATION.cff file. This file is part
+*** |  of REMIND and licensed under AGPL-3.0-or-later. Under Section 7 of
+*** |  AGPL-3.0, you are granted additional permissions described in the
+*** |  REMIND License Exception, version 1.0 (see LICENSE file).
+*** |  Contact: remind@pik-potsdam.de
+*** SOF ./modules/33_CDR/portfolio/equations.gms
+
+
+***---------------------------------------------------------------------------
+*** Equations concerning two or more options
+
+***---------------------------------------------------------------------------
+*'  CDR Final Energy Balance
+***---------------------------------------------------------------------------
+q33_demFeCDR(t,regi,entyFe)$(entyFe2Sector(entyFe,"cdr"))..
+	sum(fe2cdr(entyFe,entyFe2,te_used33),
+ 	 	v33_FEdemand(t,regi,entyFe,entyFe2,te_used33)
+	)
+  	=e=
+  	sum((entySe,te)$se2fe(entySe,entyFe,te),
+		vm_demFeSector_afterTax(t,regi,entySe,entyFe,"cdr","ETS")
+	)
+	;
+
+***---------------------------------------------------------------------------
+*'  Sum of all CDR emissions other than BECCS and afforestation, which are calculated in the core.
+***---------------------------------------------------------------------------
+q33_emiCDR(t,regi)..
+	vm_emiCdr(t,regi,"co2")
+	=e=
+	sum(te_used33, v33_emi(t,regi,te_used33))
+	;
+
+***---------------------------------------------------------------------------
+*'  Limit the amount of H2 from biomass to the demand without CDR.
+*'  It's a sustainability bound to prevent a large demand for biomass.
+***---------------------------------------------------------------------------
+q33_H2bio_lim(t,regi,te)$pe2se("pebiolc","seh2",te)..
+	vm_prodSE(t,regi,"pebiolc","seh2",te)
+	=l=
+    vm_prodFe(t,regi,"seh2","feh2s","tdh2s") - sum(fe2cdr("feh2s",entyFe2,te_used33), v33_FEdemand(t,regi,"feh2s",entyFe2,te_used33))
+	;
+
+***---------------------------------------------------------------------------
+*** DAC
+
+***---------------------------------------------------------------------------
+*'  Calculation of (negative) CO2 emissions from direct air capture.
+***---------------------------------------------------------------------------
+q33_DAC_capconst(t,regi)..
+	v33_emi(t,regi,"dac")
+	=e=
+	- sum(teNoTransform2rlf33("dac",rlf),
+		vm_capFac(t,regi,"dac") * vm_cap(t,regi,"dac",rlf)
+	)
+	;
```

Comment:
```
q33_capconst_dac(t,regi)..
	v33_emiDAC(t,regi)
	=e=
	-sum(teNoTransform2rlf_dyn33("dac",rlf2), vm_capFac(t,regi,"dac") * vm_cap(t,regi,"dac",rlf2))
	-  (1 / pm_eta_conv(t,regi,"gash2c")) * fm_dataemiglob("pegas","seh2","gash2c","cco2") * vm_otherFEdemand(t,regi,"fegas")	
	;
```

Reply:
I moved the emissions from burning `fegas` to supply DAC's heat to `q33_DAC_ccsbal`. Let's have a discussion there :)